### PR TITLE
Regionalize Nvidia download URL

### PR DESCRIPTION
**Pull request by lukeseawalker**
curl -L http://download.nvidia.com/XFree86/Linux-x86_64/418.56/NVIDIA-Linux-x86_64-418.56.run -o nvidia-commercial
curl -L http://cn.download.nvidia.com/XFree86/Linux-x86_64/418.56/NVIDIA-Linux-x86_64-418.56.run -o nvidia-china

md5sum nvidia-*
d1cf6dc33af48b53edbae5d6f31a6f89  nvidia-commercial
d1cf6dc33af48b53edbae5d6f31a6f89  nvidia-china

Signed-off-by: Luca Carrogu <carrogu@amazon.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -58,6 +58,9 @@
 default['cfncluster']['nvidia']['enabled'] = 'no'
 default['cfncluster']['nvidia']['driver_url'] = 'http://download.nvidia.com/XFree86/Linux-x86_64/418.56/NVIDIA-Linux-x86_64-418.56.run'
 default['cfncluster']['nvidia']['cuda_url'] = 'https://developer.nvidia.com/compute/cuda/10.0/Prod/local_installers/cuda_10.0.130_410.48_linux'
```
Comment (enrico-usai) [inline attributes/default.rb]: Could we do the same for cuda driver?

File: attributes/default.rb [hunk 2]
```diff
@@ -58,6 +58,9 @@
...
+if node['cfncluster']['cfn_region'].start_with?("cn-")
```
Comment (enrico-usai) [inline attributes/default.rb]: We could move the "if" closer to the driver_url initialization or maybe use an if/else block

File: attributes/default.rb (hunk 1 above)
Comment (lukeseawalker) [reply attributes/default.rb]: Tried but that same version is not present in the mirror

Comment (enrico-usai) [reply attributes/default.rb]: ok, thank you!

**Pull request merged**